src/utils/runtime/Cargo.toml

```diff
@@ -27,6 +31,7 @@ parking_lot = { version = "0.12", features = ["deadlock_detection"] }
 pprof = { version = "0.11", features = ["flamegraph"] }
 prometheus = { version = "0.13" }
 risingwave_common = { path = "../../common" }
+risingwave_variables = { path = "../variables" }
```

Comment:
nit: maybe we should put this in `common/variables` (separated crates instead of a large `common` might be good though), because: `utils` is general purpose
https://github.com/risingwavelabs/risingwave/blob/1daf42e98bd8d0a50aef154a834296414f262a00/src/README.md#L12
while `common` is for risingwave.

Reply:
... and also `mv utils/runtime common` 🤡